src/styles.scss

```diff
@@ -1 +1,61 @@
 /* You can add global styles to this file, and also import other style files */
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600%3B700%3B800&display=swap");
```

Comment:
FYI: It would be safer to download font files and attach to project but you don't need to change it